apps/browser-extension-wallet/src/lib/scripts/background/__tests__/cache-nami-metadata.test.ts

```diff
@@ -199,4 +200,113 @@ describe('cacheNamiMetadataSubscription', () => {
       })
     );
   });
+
+  it('should emit only once if UTXO and address observables emit simultaneously', () => {
```

Comment:
```suggestion
  it('should emit only once if UTxO and address observables emit simultaneously', () => {
```
Just a reminder of the style, no need to update it here

Reply:
done here https://github.com/input-output-hk/lace/pull/1546/commits/9140b12a3f4d168d34332c6e1a592ace53dbccc3